tokio/src/runtime/builder.rs

```diff
@@ -1314,6 +1329,15 @@ cfg_rt_multi_thread! {
                 Ok(Runtime::from_parts(Scheduler::MultiThreadAlt(scheduler), handle, blocking_pool))
             }
         }
+
+        fn get_spawn_concurrency_level(core_threads : usize) -> usize {
+            const MAX_SPAWN_CONCURRENCY_LEVEL: usize = 1 << 16;
+                    let mut size = 1;
+                    while size / 4 < core_threads && size < MAX_SPAWN_CONCURRENCY_LEVEL {
+                        size <<= 1;
+                    }
+                    size.min(MAX_SPAWN_CONCURRENCY_LEVEL)
+            }
```

Comment:
If we are removing it as an option from the builder, then all changes related to `spawn_concurrency_level` in this file and other files should be removed. Instead, I would just move this into `runtime/task/list.rs` as a const.